### PR TITLE
[MM-60808] Replace FormattedMarkdownMessage in 'webapp/channels/src/components/admin_console/license_settings/team_edition/team_edition_right_panel.tsx' with FormattedMessage

### PR DESCRIPTION
**Pull request by Gesare5**
#### Summary

Replaces FormattedMarkdownMessage with FormattedMessage in team_edition_right_panel.tsx 
Updates en.json

#### Ticket Link

Fixes  [#28472](https://github.com/mattermost/mattermost/issues/28472)
Jira  [https://mattermost.atlassian.net/browse/MM-60808](https://mattermost.atlassian.net/browse/MM-60808)

#### Screenshots

Screenshot4

#### Release Note

```release-note
NONE
```

### CODE REVIEW

**Comment by Gesare5** [comment]
Hey @M-ZubairAhmed , 'localizeMessage' is also used in this component(line 132) and is deprecated, so I was wondering, do I just go ahead and swap that out for FormattedMessage?

File: webapp/channels/src/components/admin_console/license_settings/team_edition/team_edition_right_panel.tsx [hunk 1]
```diff
@@ -77,21 +76,24 @@ const TeamEditionRightPanel: React.FC<TeamEditionRightPanelProps> = ({
                     </button>
                 </p>
                 <p className='upgrade-legal-terms'>
-                    <FormattedMarkdownMessage
-                        id='admin.license.enterprise.upgrade.acceptTermsInitial'
-                        defaultMessage='By clicking **Upgrade**, I agree to the terms of the Mattermost '
+                    <FormattedMessage
+                        id='teamEditionRightPanel.acceptTermsInitial'
```
Comment (M-ZubairAhmed) [inline webapp/channels/src/components/admin_console/license_settings/team_edition/team_edition_right_panel.tsx:80]: can we have the name space as `admin.licenseSettings.teamEdition.teamEditionRightPanel.xxx` in others as well

File: webapp/channels/src/components/admin_console/license_settings/team_edition/team_edition_right_panel.tsx [hunk 2]
```diff
@@ -114,8 +116,8 @@ const TeamEditionRightPanel: React.FC<TeamEditionRightPanelProps> = ({
         upgradeButton = (
             <div>
                 <p>
-                    <FormattedMarkdownMessage
-                        id='admin.license.upgraded-restart'
+                    <FormattedMessage
+                        id='teamEditionRightPanel.upgraded-restart'
```
Comment (M-ZubairAhmed) [inline webapp/channels/src/components/admin_console/license_settings/team_edition/team_edition_right_panel.tsx:120]:
```suggestion
                        id='admin.licenseSettings.teamEdition.teamEditionRightPanel.upgradedRestart'
```

**Review by M-ZubairAhmed — CHANGES_REQUESTED** [review]

**Comment by M-ZubairAhmed** [comment]
> Hey @M-ZubairAhmed , 'localizeMessage' is also used in this component(line 132) and is deprecated, so I was wondering, do I just go ahead and swap that out for FormattedMessage?

That would be much appreciated thank you. You can use the useIntl hook

File: webapp/channels/src/components/admin_console/license_settings/team_edition/team_edition_right_panel.tsx [hunk 3]
```diff
@@ -127,7 +129,7 @@ const TeamEditionRightPanel: React.FC<TeamEditionRightPanelProps> = ({
                     >
                         <LoadingWrapper
                             loading={restarting}
-                            text={localizeMessage({
+                            text={intl.formatMessage({
                                 id: 'admin.license.enterprise.restarting',
```
Comment (M-ZubairAhmed) [inline webapp/channels/src/components/admin_console/license_settings/team_edition/team_edition_right_panel.tsx:133]: can we keep the consistent ID pattern as we did above here

**Review by M-ZubairAhmed — CHANGES_REQUESTED** [review]

File: webapp/channels/src/components/admin_console/license_settings/team_edition/team_edition_right_panel.tsx [hunk 4]
```diff
@@ -37,6 +35,7 @@ const TeamEditionRightPanel: React.FC<TeamEditionRightPanelProps> = ({
     setClickNormalUpgradeBtn,
 }: TeamEditionRightPanelProps) => {
     let upgradeButton = null;
+    const intl = useIntl();
     const onHandleUpgrade = (e: React.MouseEvent<HTMLButtonElement>) => {
```
Comment (M-ZubairAhmed) [inline webapp/channels/src/components/admin_console/license_settings/team_edition/team_edition_right_panel.tsx:39]:
```suggestion
    const intl = useIntl();
    
    const onHandleUpgrade = (e: React.MouseEvent<HTMLButtonElement>) => {
```

**Review by M-ZubairAhmed — CHANGES_REQUESTED** [review]

**Review by M-ZubairAhmed — APPROVED** [review]
This looks good :shipit: Thank you

**Review by larkox — APPROVED** [review]

**Review by yasserfaraazkhan — APPROVED** [review]
tested with cloud plugin. displays as expected

**Pull request merged**